TELCODOCS:605 - cleaned up release notes document for incoming featur… (#47390)
openshift/openshift-docs: sandboxed_containers/sandboxed-containers-4.11-release-notes.adoc

:_content-type: ASSEMBLY
[id="sandboxed-containers-1-3-release-notes"]
= {sandboxed-containers-first} {sandboxed-containers-version} release notes
include::_attributes/common-attributes.adoc[]
:context: sandboxed-containers-release-notes

toc::[]

[id="sandboxed-containers-1-3-about-this-release"]
== About this release

These release notes track the development of {sandboxed-containers-first} {sandboxed-containers-version} alongside Red Hat {product-title} {product-version}.

This product is fully supported and enabled by default as of {product-title} 4.10.

[id="sandboxed-containers-1-3-new-features-and-enhancements"]
== New features and enhancements

[id="sandboxed-containers-1-3-bug-fixes"]
== Bug fixes

[id="sandboxed-containers-1-3-known-issues"]
== Known issues

[id="sandboxed-containers-1-3-asynchronous-errata-updates"]
== Asynchronous errata updates

> Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata is https://access.redhat.com/downloads/content/290/[available on the Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.

Reviewer: ```suggestion
Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata are available on the https://access.redhat.com/downloads/content/290/[Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.
```

Reviewer: Nit: where possible, link text should use a short title that distinctly identifies the link.

https://www.ibm.com/docs/en/ibm-style?topic=elements-web-ip-email-addresses#referring-to-web-addresses

also "errata" is a plural.

Reviewer: Also, use `link:LINK_URL[LINK_TEXT]` for links. See https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites

Author: This was copied directly from OCP release notes. I hadn't noticed the rogue "errata is", and it looks like the OCP team has since fixed that in their release notes as well, but the link is still structured the same there... I'll fix both on my end.

> Security, bug fix, and enhancement updates for {sandboxed-containers-first} {product-version} are released as asynchronous errata through the Red Hat Network. All {product-title} {product-version} errata are available on the https://access.redhat.com/downloads/content/290/[Red Hat Customer Portal]. See the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle] for more information about asynchronous errata.

Reviewer 2: For xrefs, please put the introductory phrase first, and then the xref itself: "For more information about asynchronous errata, see the https://access.redhat.com/support/policy/updates/openshift[{product-title} Life Cycle].

Author: Hi Lisa. All of this content was directly copied from the errata section in the OCP release notes template.

> Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.

Reviewer: ```suggestion
You can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM) on the Red Hat Customer Portal. When enabled, you are notified by email whenever new errata relevant to your registered systems are released.
```

Using a direct voice, "you/your" instead of "users" makes the text a bit easier to read.

Author: Again, copied directly from OCP RN. Gonna leave this one for now.

> Red Hat Customer Portal users can enable errata notifications in the account settings for Red Hat Subscription Management (RHSM). When errata notifications are enabled, users are notified via email whenever new errata relevant to their registered systems are released.

Reviewer 2: Use "by" instead of "via".

Author: Same comment as above.

[NOTE]
====

> Red Hat Customer Portal user accounts must have systems registered and consuming {product-title} entitlements for {product-title} errata notification emails to generate.

Reviewer: ```suggestion
You must register your systems and {product-title} entitlements in the Red Hat Customer Portal so that you can receive errata notification emails.
```

Author: Same comment as above.
